.travis.yml

```diff
@@ -1,21 +1,27 @@
-# Config file for automatic testing at travis-ci.org
-
 language: python
-
+sudo: false
 python:
   - "3.5"
   - "3.4"
   - "2.7"
 
-before_install:
-  - pip install codecov
+before_script:
+  - flake8 verification/ --max-line-length=120
```

Comment:
Flake8 uses the config stated in setup.conf, see an example here: https://github.com/Fueled/django-push-notifications/blob/master/setup.cfg

That setup file basically makes the length not a validation rule.